OpenOversight/app/commands.py

```diff
@@ -33,28 +33,50 @@
 
 
 @click.command()
+@click.option(
+    "-u",
+    "--username",
+    "supplied_username",
+    help="username for the admin account",
+)
+@click.option(
+    "-e",
+    "--email",
+    "supplied_email",
+    help="email for the admin account",
+)
+@click.option(
+    "-p",
+    "--password",
+    "supplied_password",
+    help="password for the admin account",
+)
 @with_appcontext
-def make_admin_user():
+def make_admin_user(
+    supplied_username: str | None,
+    supplied_email: str | None,
+    supplied_password: str | None,
+):
     """Add confirmed administrator account."""
     while True:
-        username = input("Username: ")
+        username = supplied_username or input("Username: ")
         user = User.by_username(username).one_or_none()
         if user:
             print("Username is already in use")
         else:
             break
 
     while True:
-        email = input("Email: ")
+        email = supplied_email or input("Email: ")
         user = User.by_email(email).one_or_none()
         if user:
             print("Email address already in use")
         else:
             break
 
     while True:
-        password = getpass("Password: ")
-        password_again = getpass("Type your password again: ")
+        password = supplied_password or getpass("Password: ")
+        password_again = supplied_password or getpass("Type your password again: ")
```

Comment:
Could we put the while loops in a `if not supplied_...` block? I'm a little worried that if an existing email/username is supplied somehow that we'll fall into an infinite loop

Reply:
That's a much better approach! I'll revise this.